src/util/dnd.cpp

```diff
@@ -143,8 +140,8 @@ bool DragAndDropHelper::allowLoadToPlayer(
     }
 
     return !isPlaying || pConfig->getValueString(
-            ConfigKey("[Controls]",
-                        "AllowTrackLoadToPlayingDeck")).toInt();
+                                        ConfigKey("[Controls]", "AllowTrackLoadToPlayingDeck"))
```

Comment:
What happens here? I recommend to switch to an easy indent rule for line brakes that does not depends on the braces in the previous line which is afaik clang-format default. This does not play well with refactoring tools, and may move broken lines far right, out of sight.
Just 4 spaces for normal indent and 8 spaces for line brakes.

Reply:
With ColumnLimit set to 0 clang-format does not insert or remove line breaks and just applies the indentation rules. This has to be fixed manually.